sdk/src/app/service/TransactionService.ts

```diff
@@ -297,7 +297,7 @@ export default class TransactionService extends Service {
 			} else if (t instanceof TransferTransaction) {
 				let message = '';
 				const tokenTransfers = (t as TransferTransaction)
-					.tokenTransfers;
+					.nftTransfers;
```

Comment:
I don't understand this change... ❓

Reply:
With the new sdk version, tokenTransfers becomes in a protected variable. We have to use nftTransfer instead.